collections/impact-observatory-lulc-map.yaml

```diff
@@ -0,0 +1,46 @@
+Name: 10m Annual Land Use Land Cover (9-class) 
+Description: |
+  The 10m Annual Land Use Land Cover (LULC) map is produced Impact Observatocy, Microsoft, and Esri collaboratively.
+  The data collection is derived from ESA Sentinel-2 imagery at 10m resolution globaly using Impact Ovservatory's state of the art deep learning AI land classification model which is trained by billions of human-labeled image pixels.
+  There are 9 LULC classes generated by the algorithm, including Built, Crops, Trees, Water, Rangeland, Flooded Vegetation, Snow/Ice, Bare Ground, and Clouds.
```

Comment:
I suggest to add links to reference the original data sources, here  in the description also